eng/pipelines/deployAPK.yml

```diff
@@ -0,0 +1,270 @@
+name: $(date:yyyyMMdd)$(rev:.r) $(Build.BuildId)

```

Comment:
Thanks @ShaunaSong  for migrating to yml.
I have updated existing pipeline yesterday to include pom file in the souces.jar file when building a package. 
Then new task created is "Put .pom file to the .aar file"